float64[7] q
---
bool success
string message
